include/sonic/dom/type.h

```diff
@@ -22,7 +22,7 @@ namespace sonic_json {
 enum TypeFlag {
   // BasicType: 3 bits
   kNull = 0,    // xxxxx000
-  kNone = 1,    // xxxxx001
+  kRaw = 1,    // xxxxx001
```

Comment:
I suggest defining kRaw as 5

Reply:
this is a basic type, are there more details?